lib/activerecord-import/import.rb

```diff
@@ -61,6 +61,7 @@ def import(*args, &block)
 
       models.each do |m|
         m.public_send "#{symbolized_foreign_key}=", owner_primary_key_value
+        m.public_send "#{reflection.type}=", owner.class if reflection.type
```

Comment:
This has been updated to do `owner.class.name`, which is the same thing that #252 does.